Makefile

```diff
@@ -126,6 +136,12 @@ push:
 		$(DOCKER) push $(REGISTRY)/$(TARGET):latest; \
 	fi
 
+	if [ "$(TRAVIS_PULL_REQUEST)" = "false" ] && [ "$(GIT_BRANCH)" = "master" ]; \
```

Comment:
You can insert this `master` branch conditional check inside the previous `TRAVIS_PULL_REQUEST` check.

Reply:
Done.